src/Operation/Aggregate.php

```diff
@@ -289,7 +292,7 @@ public function execute(Server $server)
     private function createCommand(Server $server)
     {
         $cmd = [
-            'aggregate' => $this->collectionName,
+            'aggregate' => isset($this->collectionName) ? $this->collectionName : 1,
```

Comment:
The use of "1" wasn't really documented: https://github.com/mongodb/docs/pull/3357